File: DirectoryListing/app/src/main/java/com/example/directorylisting/entities/Directory.java
package com.example.directorylisting.entities;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by Michael Steele on 3/22/17.
 * Copyright © 2017 Michael Steele. All rights reserved.
 */

public class Directory extends RealmObject {

    public RealmList<Individual> individuals;

    public Directory clear() {
        individuals = new RealmList<Individual>();
        return this;
    }

}

File: DirectoryListing/app/src/main/java/com/example/directorylisting/entities/Directory.kt
package com.example.directorylisting.entities

import io.realm.RealmList
import io.realm.RealmObject

/**
 * Created by Michael Steele on 3/22/17.
 * Copyright © 2017 Michael Steele. All rights reserved.
 */

open class Directory : RealmObject() {

    var individuals: RealmList<Individual> = RealmList<Individual>()

    fun clear(): Directory {
        individuals = RealmList()
        return this
    }

}
